tools/tasks/gulp-pa11y.js

```diff
@@ -0,0 +1,92 @@
+const fs = require('fs');
+const path = require('path');
+
+const pa11y = require('pa11y');
+const reporter = require('pa11y-reporter-cli'); // As pa11y 5 stabilizes, we can pull this off pa11y
+const _ = require('lodash');
+const through = require('through2');
+
+// Grab project root pa11y.json as object
+const pallyJson = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'pa11y.json')));
+// Custom logging tools for pa11y
+const logConfig = {
+  log: {
+    debug: console.log,
+    error: console.error,
+    info: console.log,
+  },
+};
+// Smash them together
+const pa11yConfig = _.assign(pallyJson, logConfig);
+
+/**
+ * Run pa11y testing one after the next
+ * @returns {*}
+ */
+function pa11yConcurrent() {
+  function bufferContents(vinylFile, encoding, callback) {
+    // ignore empty files
+    if (vinylFile.isNull()) {
+      callback();
+      return;
```

Comment:
That totally seems reasonable, does something else report in on the obvious build problem of files showing up empty?

Reply:
Nope, this is just gulp's `gulp.src()` mechanism. If a file exists, it gets sent in regardless. I'm sure there's some YAGP (Yet Another Gulp Plugin) that could let use do this instead:

```js
gulp.task('test:accessibility', () => gulp
  .src([
    'dist/public/patterns/**/*.html',
    ...
  ])
  .pipe(ignoreEmpty()) // This plugin probably exists
  .pipe(pa11yConcurrent()));
```

Reply:
Part of my point is that since empty files at the end of a pipeline are obviously an error of some kind, shouldn't we flag those with a warning? Or does gulp do that already?

Reply:
I don't really think empty files at the end of a pipeline is technically an error. That `isNull()` check is being provided every single file in a glob so it's rational to assume some files will be empty and to just skip them.